data-gen/gen-autoconf.py

```diff
@@ -1,35 +1,50 @@
-#!/usr/bin/python
-import sys
-import os
+#!/usr/bin/env python3
```

Comment:
why via ```/usr/bin/env```?

Reply:
So it can live in a venv, and use the shim python3 from in there, that is how I tested it.

Reply:
which in turn means for the package to be done, this needs to be sed'ed again to be ```/usr/bin/python3``` - packages are supposed to use the system python interpreter, not any random one in $path

Reply:
That makes sense. Do we have an rpmlint that detect this?

Reply:
yes - but like many others it's being ignored by most packagers

An overview of current violators is at
 https://rpmlint.opensuse.org/openSUSE:Factory/x86_64/standard?rule=env-script-interpreter

Reply:
Ok, done.